### PR TITLE
Quizap game foundation

### PR DESCRIPTION
**Pull request by greypants**
This PR builds on a socket.io example chat app that i'd refactored into a React App. All these commits are replacing that starting point with a quiz app (Quizap!—I'm open to better names :sweat_smile: ) demo I'm building for BDConf. The UI/UX/Design/Code is still super in progress, and far from finished.

The goal is to sell how powerful a browser-based app can be through a live audience participation demo.
- [x] Use WebAudio API
- [x] Show how WebSockets can be used to bring a digital experience into a real life experience (outside your screen, interacting with the environment and people around you)
- [ ] Show some 60fps animations and transitions, maybe some physics
- [ ] Use motion sensors and touch
#### More to-dos
- [ ] Only the first person to buzz in should get a sound effect and an active status
- [ ] Everyone else gets a "too slow" status, showing (# of # zappers). 
- [ ] If Zapper #1 gets the question wrong, zapper 2 should be activated and given a chance to answer
- [ ] Decrement player count on disconnect
- [ ] Create rooms
## [Moderator Screen Demo](https://quizap-pr-1.herokuapp.com/#/moderator)

[image](https://quizap-pr-1.herokuapp.com/#/moderator)

A moderator can:
- Activate the game
- See "Yep/Nope" buttons **when someone's buzzed (zapped) in.** (do this as a player in another browser window)
- Accept or Reject their answer
## [Player Screen Demo](https://quizap-pr-1.herokuapp.com/#/moderator)

[image](https://quizap-pr-1.herokuapp.com/#/moderator)

A Player can:
- Buzz in (Zap)
- Receive game status (active, yep, nope, ready)

### CODE REVIEW

File: client/js/components/shared/StatusLight.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default React.createClass({
+  propTypes: {
+    status: React.PropTypes.string.isRequired,
```
Comment (nhunzaker) [inline client/js/components/shared/StatusLight.js:5]: Really required? Could you do something with `classNames` or a default status?

File: client/sass/base/_animations.sass [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@keyframes spin
+  0%
+    transform: rotate(0deg)
+
+  100%
+    transform: rotate(359deg)
```
Comment (nhunzaker) [inline client/sass/base/_animations.sass:6]: Curious why `e` is highlighted here.

**Comment by nhunzaker** [comment]
:+1:  Awesome!

File: client/js/components/shared/StatusLight.js (hunk 1 above)
Comment (greypants) [inline client/js/components/shared/StatusLight.js:5]: oh good call. It should have a default status.

**Pull request merged**